benchmarks/common/db.go

```diff
@@ -18,12 +18,13 @@ package common
 import (
 	"database/sql"
 	"fmt"
-	"github.com/cossacklabs/acra/benchmarks/config"
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/sirupsen/logrus"
 	"os"
 )
 
 func connect(connectionString string) *sql.DB {
-	db, err := sql.Open("postgres", connectionString)
+	db, err := sql.Open("pgx", connectionString)
```

Comment:
So we switch to a different PgSQL driver? It is better in terms of performance?

Reply:
there are occurred errors with the previous driver with inserting binary data. plus this driver is stopped support and live only in maintenance mode. Pgx is pure golang driver that suggested by https://github.com/lib/pq driver ("postgres").